components/search/MoreFacetsDropdown.vue

```diff
@@ -45,7 +56,28 @@
         type: Array,
         default: () => []
       }
+    },
+
+    data() {
+      return {
+        selected: {}
+      };
+    },
+
+    computed: {
+      selectedAmount() {
+        let sum = 0;
```

Comment:
This whole function can be reduced to just:
```js
return Object.values(this.selected).flat().length;
```

Reply:
Sadly the flat() method doesn't work in IE Edge  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat#Browser_compatibility

Reply:
How about this then:
```js
return [].concat(...Object.values(this.selected)).length;
```

Reply:
Done